test/dox/seqan3_doxygen_cfg.in

```diff
@@ -0,0 +1,42 @@
+PROJECT_NAME           = "SeqAn3"
+
+## PATHS
+OUTPUT_DIRECTORY       = ${SEQAN3_DOXYGEN_OUTPUT_DIR}
+INPUT                  = ${SEQAN3_DOXYGEN_INCLUDE_DIR}
+STRIP_FROM_PATH        = ${SEQAN3_DOXYGEN_INCLUDE_DIR}
+STRIP_FROM_INC_PATH    = ${SEQAN3_DOXYGEN_INCLUDE_DIR}
+EXAMPLE_PATH           = ${SEQAN3_DOXYGEN_INCLUDE_DIR}/../
+
+## DOT SUPPORT
+HAVE_DOT               = ${SEQAN3_DOXYGEN_HAVE_DOT}
+DOT_IMAGE_FORMAT       = svg
+INTERACTIVE_SVG        = ${SEQAN3_DOXYGEN_HAVE_DOT}
+
+## MISC OPTIONS
+RECURSIVE              = YES
+DISABLE_INDEX          = YES
+GENERATE_TREEVIEW      = YES
+INLINE_INHERITED_MEMB  = YES
+COLLABORATION_GRAPH    = NO
+TEMPLATE_RELATIONS     = YES
+ENABLE_PREPROCESSING   = YES
+MACRO_EXPANSION        = YES
+EXPAND_ONLY_PREDEF     = YES
+SORT_BRIEF_DOCS        = YES
+
+PREDEFINED             = "CEREAL_SERIALIZE_FUNCTION_NAME=serialize" \
+                         "CEREAL_LOAD_FUNCTION_NAME=load" \
+                         "CEREAL_SAVE_FUNCTION_NAME=save" \
+                         "CEREAL_LOAD_MINIMAL_FUNCTION_NAME=load_minimal" \
+                         "CEREAL_SAVE_MINIMAL_FUNCTION_NAME=save_minimal" \
+                         "SEQAN3_DOXYGEN_ONLY(x)= x" \
+                         "${SEQAN3_DOXYGEN_PREDEFINED_NDEBUG}"
+
+## detect headers without extensions (in std module)
+EXTENSION_MAPPING      = .no_extension=C++
+FILE_PATTERNS          = *
+                         
+## Developer VS user mode
+EXTRACT_PRIVATE        = ${SEQAN3_DOXYGEN_EXTRACT_PRIVATE}
+ENABLED_SECTIONS       = ${SEQAN3_DOXYGEN_ENABLED_SECTIONS}
+EXCLUDE_SYMBOLS        = ${SEQAN3_DOXYGEN_EXCLUDE_SYMBOLS}
```

Comment:
Can we somehow do this centrally somehwere? I am worried, now we have it in
1. SeqAn-Repo
1. SeqAn-Infrastructure Repo
1. SeqAn-Wiki

They are bound to become out-of-sync :cry:

Reply:
I think here in the repo would be the best place.